kategory/src/main/kotlin/kategory/data/EitherT.kt

```diff
@@ -78,3 +78,51 @@ data class EitherT<F, A, B>(val MF: Monad<F>, val value: HK<F, Either<A, B>>) :
         return GA.map(fa, { EitherT(MF, MF.map(it.lower(), { it.ev() })) })
     }
 }
+
+class EitherTInstances<F, L>(val MF : Monad<F>) : EitherTMonadError<F, L> {
+    override fun MF(): Monad<F> = MF
```

Comment:
This class requires a inline fun invoke constructor in the companion object that defaults the MF monad.